tests/test_random.py

```diff
@@ -49,3 +50,36 @@ def test_random_check_otput(func):
     for i in range(res.size):
         assert res[i] >= 0.0
         assert res[i] < 1.0
+
+
+def test_randn_normal_distribution():
+    pts = 1000
+    alpha = 0.05
+    dpnp.random.seed(28041990)
+    x = dpnp.random.randn(pts)
+    _, p = stats.normaltest(x)

```

Comment:
How it is works is `stats` commented out above?

Reply:
Added comments